### PR TITLE
GSdx: Add missing CRC id.

### PR DESCRIPTION
**Pull request by Al0ner**
GSdx: Add missing CRC id.
Add TombRaiderUnderworld RU.

### CODE REVIEW

File: plugins/GSdx/GSCrc.cpp [hunk 1]
```diff
@@ -136,8 +136,7 @@ CRC::Game CRC::m_games[] =
 	{0xB01A4C95, ICO, JP, 0},
 	{0x2DF2C1EA, ICO, KO, 0},
 	{0x5C991F4E, ICO, EU, 0},
-	{0x7ACF7E03, ICO, NoRegion, 0}, // same CRC as {SpyroNewBeginning, NoRegion}
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Why did you remove this ?

Comment (Al0ner) [reply plugins/GSdx/GSCrc.cpp]: It does'nt make any sense for this game.
There is no one original PS2 game with this ID.
But almost all unofficial RU-versions of games with a multiloader - have exactly this CRC-id.
7ACF7E03 - is the ID of the multiloader, not the executable file of the game.
Therefore, if you repacked such an image, delete the multiloader and leave only the executable file of the game - the CRC-id will change to the original one, or at least to the unique one.
You know that if the same CRC-hacks and WD-pnaches be applied to different games - it can break them. Therefore, there is no sense in this ID.

Comment (lightningterror) [reply plugins/GSdx/GSCrc.cpp]: Ah yes you're right, I did a bit of research and see multiple games with the same crc.

In that case you could also remove line 414 and 415.
These to be precise.
```
	// {0x7ACF7E03, SpyroNewBeginning, NoRegion, 0}, // same CRC as {ICO, NoRegion}
	// and as "Twisted Metal - Black" (PAL).
```

**Comment by lightningterror** [comment]
Next time please squash your commits properly with Git Shell.
